### DISCUSSION TITLE
SM2算法，导入PEM格式私钥出错

### DISCUSSION THREAD

**Discussion by songjinlian** [Leon406/ToolsFx#248] [Q&A] [answered]
出错截图：
![图片](https://github.com/Leon406/ToolsFx/assets/101912268/f9540a73-6448-48fa-bb66-8fdc0acaf78d)
手工删除“-----BEGIN EC PRIVATE KEY-----和-----END EC PRIVATE KEY-----
”就正确了：
![图片](https://github.com/Leon406/ToolsFx/assets/101912268/a3c71e14-2e6c-45b0-bcd8-8d06966130b2)

**Comment by Leon406** [comment, marked as answer]
感谢反馈，已优化解析，构建包见最新的github action
